user-storage-jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -5,7 +5,7 @@
         http://java.sun.com/xml/ns/persistence
         http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
     <persistence-unit name="user-storage-jpa-example">
-        <jta-data-source>java:jboss/datasources/ExampleDS</jta-data-source>
+        <jta-data-source>java:jboss/datasources/ExampleXADS</jta-data-source>
```

Comment:
There's not an ExampleXADS registered is there?

Reply:
AFAIK the _ExampleXADS_ is registered, when the user add datasource by maven profile `add-datasource`. Everything is described in README. This datasource is related only for this QS, because it has to be a `xa` datasource and _KeycloakDS_ is default `non-xa` DS and ExampleDS is `non-xa` too.